src/protocol/ldap/org/apache/jmeter/protocol/ldap/sampler/LDAPExtSampler.java

```diff
@@ -660,7 +670,7 @@ private void bindOp(SampleResult res) throws NamingException {
         }
         try {
             res.sampleStart();
-            ctx = LdapExtClient.connect(getServername(), getPort(), getRootdn(), getUserDN(), getUserPw(),getConnTimeOut(),isSecure());
+            ctx = LdapExtClient.connect(getServername(), getPort(), getRootdn(), getUserDN(), getUserPw(),getConnTimeOut(),isSecure(),isTrustAll());
```

Comment:
Again spacepolice time :) plus you could add a line break to shorten the lines a bit.

Reply:
yup will do.